src/components/table.tsx

```diff
@@ -37,30 +44,83 @@ function Row({
   );
 }
 
-function Table({ data, numberFormatter = String }: TableProps): VNode {
+/**
+ * Used in textContent, created once to save some overhead
+ */
+const doc = document.implementation.createHTMLDocument("");
+
+function textContent(v: VNode): string {
+  render(v, doc.body);
```

Comment:
It might be okay, but I'm not 100% certain that this won't try diffing the new node against the previous call's body contents.

I wanna say that the [third render argument](https://preactjs.com/guide/v10/api-reference/#render) could be used to trick it into ignoring the current state?

```suggestion
  render(v, doc.body, true);
```

Or could clear it out ahead:

```suggestion
  doc.body.innerHTML = "";
  render(v, doc.body);
```

Or just leave it as is and let diffing do its thing 🤷

Reply:
the `innerHTML` trick worked better: 67351eb6c7a0fedd6931024257670eb6fadc8bd9